dynamics-nav-app/design-details-item-application.md

```diff
@@ -1,5 +1,5 @@
 ---
-    title: Design Details - Item Application 
+    title: Item Application for Design Details 
```

Comment:
I think the way some of these titles are worded can be confusing...I know you're trying to remove duplicate titles, but this reads to me as if Design Details were a software and you talk about the Item Application within. If that is the case, it makes sense! If not, you could consider changing it. Just something to consider!

Reply:
Gotcha

Reply:
Please be careful here. "Design Details" was conceived as a category of articles many years ago, and the articles' title and H1 start with those two words. So the principle here is "design details about item application", where "item application" is an industry-widely understood term.

Reply:
@edupont04 Thanks for the clarification!